src/odemis/driver/xt_client.py

```diff
@@ -1275,13 +1275,24 @@ def start_autostig(self, n_images=17, sweep_range=0.015, dwell_time=3e-6, resolu
 
         Returns
         -------
-        Position
+        optimal_stigmation
             Optimal stigmator setting.
         """
         with self._proxy_access:
             self.server._pyroClaimOwnership()
-            return self.server.start_autostig(n_images, sweep_range, dwell_time, resolution,
-                                              reduced_area_size, save_results, results_fp)
+            self.server._pyroTimeout = 300  # seconds
+            try:
+                optimal_stigmation = self.server.start_autostig(n_images, sweep_range, dwell_time, resolution,
```

Comment:
Could you add in the docstring, that this is blocking (and maybe also imply that every other call is blocked).